management/backup.py

```diff
@@ -57,6 +57,8 @@ def parse_line(line):
 		"--archive-dir", backup_cache_dir,
 		"--gpg-options", "--cipher-algo=AES256",
 		"--log-fd", "1",
+		"--ssh-options='-i /root/.ssh/id_rsa_miab'",
```

Comment:
Could you put the six times these two lines occur into a function, so that it's not repeated?

If there are other command-line arguments that also appear in all six locations, please move those into the function too.